Sources/Features/AppFeature/App+Reducer.swift

```diff
@@ -91,8 +92,6 @@ public struct App: Sendable, FeatureReducer {
 	public func reduce(into state: inout State, viewAction: ViewAction) -> Effect<Action> {
 		switch viewAction {
 		case .task:
```

Comment:
N.B.! this code in `task` is NEVER run, since `task` is never run. Read section `Important Discovery` in description of this PR.